addon/components/o-s-s/infinite-select.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="upf-infinite-select" {{did-insert this.onRender}} ...attributes>
+<div class="fx-col upf-infinite-select {{if (not this.inline) 'upf-infinite-select--absolute'}}" {{did-insert this.onRender}} ...attributes>
```

Comment:
isn't it better to do `if inline -> upf-infinite-select--inline` to:
1) avoid the negations
2) don't break the existing because right now, if we bump oss independently, it will break existing :( 
3) keep the current version as default